system/Test/bootstrap.php

```diff
@@ -29,9 +29,7 @@
 defined('HOMEPATH') || define('HOMEPATH', realpath(rtrim(getcwd(), '\\/ ')) . DIRECTORY_SEPARATOR);
 $source = is_dir(HOMEPATH . 'app')
     ? HOMEPATH
-    : (is_dir('vendor/codeigniter4/framework/')
-        ? 'vendor/codeigniter4/framework/'
-        : 'vendor/codeigniter4/codeigniter4/');
+    : (is_dir('vendor/codeigniter4/framework/') ? 'vendor/codeigniter4/framework/' : 'vendor/codeigniter4/codeigniter4/');
```

Comment:
multiline nested ternary skipped in latest rector/rector:dev-main https://github.com/rectorphp/rector-src/pull/2902 while still ok in cs-fixer check, so I still apply the change.